libs/langchain/langchain/schema/runnable/base.py

```diff
@@ -119,6 +122,24 @@ def output_schema(self) -> Type[BaseModel]:
             self.__class__.__name__ + "Output", __root__=(root_type, None)
         )
 
+    def config_schema(
+        self, *, include: Optional[Sequence[str]] = None
```

Comment:
Could you explain where this comes into play? This looks like the change I needed for remote runnables, but for that case I'm not sure I would've wanted the config defined as a method on runnables since config doesn't vary by runnable

Reply:
It will make sense in the next PR :) https://github.com/langchain-ai/langchain/pull/11282